src/main/java/Commands/ListCommand.java

```diff
@@ -0,0 +1,28 @@
+package Commands;
+
+import Main.Ui;
+
+public class ListCommand extends Command {
+    String userInput;
+
+    public ListCommand(String userInput) {
+
+        this.userInput = userInput;
+    }
+
+    /**
+     * Displays items in list via method in storelist class
+     *
+     * @return
+     */
+    @Override
+    public String execute() {
+        return storeList.displayItems();
+    }
+
+    @Override
+    public boolean isExit() {
+
+        return false;
+    }
```

Comment:
Should there be a line between line 24 and 26? Wouldn't the code look better if tge empty line is removed